crates/client/src/fuzzer/program_test_client_blocking.rs

```diff
@@ -10,12 +10,16 @@ use solana_sdk::{
 use spl_token::state::Mint;
 use tokio::runtime::Builder;
 
+use crate::client_stats::{AccumulatedStats, Stats, StatsLogger};
+use crate::constants::*;
 use crate::data_builder::FuzzClient;
 use crate::error::*;
 
 pub struct ProgramTestClientBlocking {
     ctx: ProgramTestContext,
     rt: tokio::runtime::Runtime,
+    stats: Stats,
```

Comment:
The stats should not be part of the client implementation. Now the `run_fuzzer` method imposes a trait bound `FuzzClient + StatsLogger` on the client parameter and therefore a custom client implementation would need to implement also the `StatsLogger` trait which is not desirable. It would be better to pass the stats as separate parameter.